#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x0000000104f7bfa0, pid=9452, tid=34051
#
# JRE version: OpenJDK Runtime Environment Zulu15.46+17-CA (15.0.10+5) (build 15.0.10+5-MTS)
# Java VM: OpenJDK 64-Bit Server VM Zulu15.46+17-CA (15.0.10+5-MTS, mixed mode, tiered, compressed oops, g1 gc, bsd-aarch64)
# Problematic frame:
# C  [python3.8+0x1cbfa0]  collect+0x124
#
# No core dump will be written. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# If you would like to submit a bug report, please visit:
#   http://www.azul.com/support/
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#

---------------  S U M M A R Y ------------

Command Line: -Dfile.encoding=UTF8 -ea -Xmx1024m 

Host: Mac14,6 arm64 1 MHz, 12 cores, 32G, Darwin 23.0.0
Time: Fri Dec  1 15:31:32 2023 KST elapsed time: 0.084909 seconds (0d 0h 0m 0s)

---------------  T H R E A D  ---------------

Current thread (0x0000000155c2d600):  JavaThread "Python Reference Queue" daemon [_thread_in_native, id=34051, stack(0x00000002d7658000,0x00000002d785b000)]

Stack: [0x00000002d7658000,0x00000002d785b000],  sp=0x00000002d785a7b0,  free space=2057k
Native frames: (J=compiled Java code, A=aot compiled Java code, j=interpreted, Vv=VM code, C=native code)
C  [python3.8+0x1cbfa0]  collect+0x124
C  [python3.8+0x1cd5b4]  collect_with_callback+0x4c
C  [python3.8+0x1cd538]  PyGC_Collect+0x60
C  [_jpype.cpython-38-darwin.so+0x1cd70]  JPGarbageCollection::triggered()+0x38
C  [_jpype.cpython-38-darwin.so+0x29080]  Java_org_jpype_ref_JPypeReferenceNative_wake+0x1c
j  org.jpype.ref.JPypeReferenceNative.wake()V+0
j  org.jpype.ref.JPypeReferenceQueue$Worker.run()V+42
j  java.lang.Thread.run()V+11 java.base@15.0.10
v  ~StubRoutines::call_stub
V  [libjvm.dylib+0x3e1338]  JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, Thread*)+0x324
V  [libjvm.dylib+0x3e0410]  JavaCalls::call_virtual(JavaValue*, Klass*, Symbol*, Symbol*, JavaCallArguments*, Thread*)+0xf8
V  [libjvm.dylib+0x3e04d8]  JavaCalls::call_virtual(JavaValue*, Handle, Klass*, Symbol*, Symbol*, Thread*)+0x64
V  [libjvm.dylib+0x48eae0]  thread_entry(JavaThread*, Thread*)+0xbc
V  [libjvm.dylib+0x8cb6ac]  JavaThread::thread_main_inner()+0xa0
V  [libjvm.dylib+0x8c8ba8]  Thread::call_run()+0x88
V  [libjvm.dylib+0x711f3c]  thread_native_entry(Thread*)+0x1d0
C  [libsystem_pthread.dylib+0x7034]  _pthread_start+0x88

Java frames: (J=compiled Java code, j=interpreted, Vv=VM code)
j  org.jpype.ref.JPypeReferenceNative.wake()V+0
j  org.jpype.ref.JPypeReferenceQueue$Worker.run()V+42
j  java.lang.Thread.run()V+11 java.base@15.0.10
v  ~StubRoutines::call_stub

siginfo: si_signo: 11 (SIGSEGV), si_code: 2 (SEGV_ACCERR), si_addr: 0x0000000169059e48

Register to memory mapping:

 x0=0x000000016b669870 points into unknown readable memory: 0x00000001050b0ef0 | f0 0e 0b 05 01 00 00 00
 x1=0x000000016b669870 points into unknown readable memory: 0x00000001050b0ef0 | f0 0e 0b 05 01 00 00 00
 x2=0x0000000000000002 is an unknown value
 x3=0x0 is NULL
 x4=0x00000d4d00000000 is an unknown value
 x5=0x0000a30300000000 is an unknown value
 x6=0x0000000000000001 is an unknown value
 x7=0x0 is NULL
 x8=